gulpfile.js

```diff
@@ -0,0 +1,25 @@
+var gulp = require('gulp');

```

Comment:
look for other gulp tasks which must be necessary for the application

Reply:
sure